x-pack/plugins/cloud_security_posture/public/components/csp_inline_description_list.tsx

```diff
@@ -6,12 +6,41 @@
  */
 
 import React from 'react';
-import { EuiDescriptionList, useEuiTheme, type EuiDescriptionListProps } from '@elastic/eui';
+import { EuiDescriptionList, useEuiTheme, EuiIcon, EuiCopy } from '@elastic/eui';
+import type { EuiDescriptionListProps } from '@elastic/eui';
+import { css } from '@emotion/react';
 
-const getModifiedTitlesListItems = (listItems: EuiDescriptionListProps['listItems']) =>
+const CopyButton = ({ copyText }: { copyText: string }) => (
+  <EuiCopy textToCopy={copyText}>
+    {(copy) => (
+      <EuiIcon
+        css={css`
+          :hover {
+            cursor: pointer;
+          }
+        `}
+        onClick={copy}
+        type="copy"
+      />
+    )}
+  </EuiCopy>
+);
+
+const getModifiedTitlesListItems = (listItems?: EuiDescriptionListProps['listItems']) =>
   listItems
     ?.filter((item) => !!item?.title && !!item?.description)
-    .map((item) => ({ ...item, title: `${item.title}:` }));
+    .map((item) => ({
+      ...item,
+      title: `${item.title}:`,
+      description:
+        typeof item.description === 'string' ? (
+          <span>
```

Comment:
Is this showing a React key warning?

Reply:
I'm not sure I follow Sean, is this regarding the CSS which i ended up not implementing in the end? let's take this offline so i can understand better